design/Perception/Perception.md

```diff
@@ -175,7 +175,8 @@ Need to fill `lamp_states` in `autoware_traffic_light_msg::TrafficLightState.msg
 
 | Property    | Definition                                                      | Data Type                               | Parent Data Type                              |
 | ----------- | --------------------------------------------------------------- | --------------------------------------- | --------------------------------------------- |
-| lamp_states | Seguence of traffic light result from the closest traffic light | `autoware_perception_msgs::LampState[]` | `autoware_perception_msgs::TrafficLightState` |
+| lamp_states | Seguence of traffic light result                                | `autoware_perception_msgs::LampState[]` | `autoware_perception_msgs::TrafficLightState` |
```

Comment:
There is a typo.
Seguence -> Sequence

Reply:
Thank you for finding typo. I'll modify it.